common/src/main/java/com/wynntils/functions/CharacterFunctions.java

```diff
@@ -49,7 +49,7 @@ public Integer getValue(FunctionArguments arguments) {
         }
 
         @Override
-        public List<String> getAliases() {
```

Comment:
I think it probably would have been better to keep the getAliases name for what the Functions override, and then introduced a new function for the caching version in the superclass. My arguments are that there are many subclass implementation, but just one or a few calls to the superclass caching version, so it will be less unnecessary changes to the code base. And secondly, I think `getAliases` sounds slightly better when the functions override it.

But if you disagree, you can keep it as it is as well. I don't feel too strongly about it.

Reply:
I still want to make it protected.. but I changed the names.